src/ert/ensemble_evaluator/_builder/_realization.py

```diff
@@ -1,52 +1,52 @@
 import logging
-from typing import Optional, Sequence
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Optional, Sequence
 
 from typing_extensions import Self
 
-from ._step import LegacyJob, LegacyStep
+from ert.config.ext_job import ExtJob
 
+if TYPE_CHECKING:
+    from ert.run_arg import RunArg
 SOURCE_TEMPLATE_REAL = "/real/{iens}"
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class LegacyJob:
```

Comment:
I think we should make an issue for removing all these `Legacy` prefixes.

Reply:
I regard it as part of #6218